src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -42,27 +42,29 @@ public double getValue() {
     }
 
     /**
-     * Fetch a {@link Association} by its value.
+     * Fetch a {@link Association} by its exact value.
      *
      * @param value The friendliness value
      * @return The association related to the friendliness value. <code>null</code> if there is no {@link Association} with the provided value.
      * @author 086
      */
-    public static Association byValue(double value) {
+    public static Association byExactValue(final double value) {
         return Arrays.stream(values()).filter(association -> association.getValue() == value).findAny().orElseGet(null);
     }
 
     /**
-     * Fetch the {@link Association} closest to a friendliness value.
-     * <p>Edge cases: <code>0.5</code> becomes {@link Association#ALLY} and <code>-0.5</code> becomes {@link Association#ENEMY}</p>
+     * Fetch the {@link Association} convert value to association.
+     * No matter how friendly someone is as long as he is friendly he gets the ally association. The same goes for enemies.
+     * So values below 0 are enemies and above are allies.
+     *
      * @param value The friendliness value
-     * @return      The {@link Association} with a friendliness value closest to the provided value
-     * @author 086
+     * @return      The {@link Association} with a friendliness value depending to the provided value
+     * @author IronException
      */
-    public static Association closestToValue(double value) {
-        if (value == Double.MIN_VALUE) return Association.ENEMY;
-        if (value == Double.MAX_VALUE) return Association.ALLY;
-        return Arrays.stream(values()).min(Comparator.comparingDouble(a -> Math.abs(a.value - value))).get();
+    public static Association byValue(final double value) {
+        if (value < 0) return Association.ENEMY;
+        if (value > 0) return Association.ALLY;
```

Comment:
Instead of `0`, use `Standard.STANDARD_NEUTRAL`

Reply:
true. I noticed that later but haven't fixed it there. Thanks for pointing out :)